Formula/go.rb

```diff
@@ -74,6 +85,19 @@ def install
     bin.install_symlink libexec/"bin/godoc"
   end
 
+  def caveats
+    s = ""
+
+    if Hardware::CPU.arm?
+      s += <<~EOS
+        This is a beta version of the Go compiler for Apple Silicon
+        (go 1.16beta1).
```

Comment:
```suggestion
        (go1.16beta1).
```

nit: it's `go1.16beta1` (like in `go version` output) or `Go 1.16beta1`.

Reply:
Will do